src/main/java/org/ice4j/socket/MultiplexingDatagramSocket.java

```diff
@@ -274,6 +384,26 @@ public MultiplexedDatagramSocket getSocket(
     }
 
     /**
+     * Adds a specific {@link MultiplexedDatagramSocket} to this instance. The
+     * given socket must be already associated with this instance.
+     * @param multiplexed the socket to add.
+     * @throws IllegalArgumentException if the given socket is not associated
+     * with this instance, or its filter is already used for another multiplexed
+     * socket.
+     */
+    public void addSocket(MultiplexedDatagramSocket multiplexed)
```

Comment:
Because I want to override the receive() method of the MultiplexedDatagramSocket from code in the bridge. I guess I could expose a callback somehow, but this was the easiest solution I found.